test/test_ops.py

```diff
@@ -496,6 +497,7 @@ def expected_fn(self, x, weight, offset, mask, bias, stride=1, padding=0, dilati
         out += bias.view(1, n_out_channels, 1, 1)
         return out
 
+    @lru_cache(maxsize=None)
```

Comment:
Deciding on caching random data is not that simple IMO. There are pros/cons of that...

Reply:
I agree there are pros/cons. Note that this is something we do on tests only and seems to help us speed-wise. I don't have a strong opinion about keeping it if it causes issues.